core/src/main/java/com/avast/bytes/UnsignedBytesComparator.java

```diff
@@ -0,0 +1,31 @@
+package com.avast.bytes;
+
+import java.nio.ByteBuffer;
+import java.util.Comparator;
+
+/**
+ * Natural Bytes comparator that order Bytes as array of unsigned bytes.
```

Comment:
What does the word "natural" means? I see that both the comparators are described as "natural" and I wonder whether that's correct.